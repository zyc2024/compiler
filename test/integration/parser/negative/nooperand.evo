int main() {
    int k = 9 + ;
}