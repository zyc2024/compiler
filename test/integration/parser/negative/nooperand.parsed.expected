2:17 error:unexpected ';'